plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergAbfsConnectorSmokeTest.java

```diff
@@ -86,8 +86,10 @@ protected QueryRunner createQueryRunner()
                                 .put("iceberg.catalog.type", "HIVE_METASTORE")
                                 .put("hive.metastore.uri", hiveHadoop.getHiveMetastoreEndpoint().toString())
                                 .put("hive.metastore.thrift.client.read-timeout", "1m") // read timed out sometimes happens with the default timeout
-                                .put("hive.azure.abfs-storage-account", account)
-                                .put("hive.azure.abfs-access-key", accessKey)
+                                .put("fs.hadoop.enabled", "false")
+                                .put("fs.native-azure.enabled", "true")
+                                .put("azure.auth-type", "ACCESS_KEY")
+                                .put("azure.access-key", accessKey)
                                 .put("iceberg.register-table-procedure.enabled", "true")
                                 .put("iceberg.writer-sort-buffer-size", "1MB")
                                 .buildOrThrow())
```

Comment:
pls add an integration test containing special characters for the partitions in `BaseIcebergConnectorSmokeTest`

Reply:
Even if it works the same with and without changes in this PR? What's the value of such a test?